src/gtk/dataview.cpp

```diff
@@ -960,21 +960,22 @@ wxgtk_tree_model_drag_data_get (GtkTreeDragSource *drag_source,
     if ( wxtree_model->stamp == 0 )
         return FALSE;
 
+    return TRUE;
```

Comment:
This was probably unintentional?

Reply:
Nope it was deliberate, if a little lazy :)

Based on what I did to get D'n'D working using PyGObject GTK, it seems that the `->drag_data_received` & `->row_drop_possible` function pointers have to be filled in with stubs even though the actual work is done thru a different interface.

Reply:
Unfortunately I still don't understand what's going on here... This callback is used as `drag_data_get` of `drag_source_iface_info`, so it really looks like it ought to do something instead of just returning `TRUE`. Or, alternatively, maybe we shouldn't define/use this interface at all?

@paulcor Would you have any idea about what's going on here by chance?

Reply:
I'm not entirely sure myself. I _suspect_ that leaving it undefined makes GTK fall back on an internal default that returns `FALSE`, which in turn vetoes emitting of the `drag-data-get` signal that `gtk_dataview_drag_data_get_callback` connects to. I'm rewriting in C one of my PyGObject utilities that uses Gtk.TreeView in order to have a closer look.